package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.2.5",
+  "version": "0.2.9",
```

Comment:
Latest release is 1.0.0, so this should be a pre-release version and > 1.0.0

Reply:
addressed